### PR TITLE
Gui improvements 29 06 23

### PR DESCRIPTION
**Pull request by maryvictol**
This PR provides updates according to UI changes
- new Actions menu for Object Storage  page
- run task `GridEngineAutoscaling` has been changed to `SGEAutoscaling`
- update NAT Gateway form locators
- add the possibility to select region at opening SSH
-  #3303

### CODE REVIEW

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/LogAO.java [hunk 1]
```diff
@@ -86,6 +87,16 @@ public LogAO waitForSshLink() {
         return this;
     }
 
+    private void openRegionSelector() {
+            get(SSH_LINK).parent().find(className("ultizone-url__expander")).click();
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/LogAO.java]: Please correct the indentation

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/NATGatewayAO.java [hunk 2]
```diff
@@ -15,6 +15,8 @@
  */
 package com.epam.pipeline.autotests.ao;
 
+import com.codeborne.selenide.Condition;
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/NATGatewayAO.java]: Unused import

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/SshPermissionTest.java [hunk 3]
```diff
@@ -129,7 +129,6 @@ public void checkAccessToSSHByDirectLinkForNonOwnerUser() {
                 .switchToMyProfile()
                 .validateUserName(user.login);
         ShellAO.open(sshLink)
-                .waitUntilTextAppears(getLastRunId())
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/SshPermissionTest.java:132]: Why did we remove this check?

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/SystemLoggingTest.java [hunk 4]
```diff
@@ -90,12 +90,12 @@ public void userAuthentication() {
             if (impersonateMode()) {
                 systemLogsAO.filterByMessage("impersonation");
             }
-            adminInfo = getInfo(systemLogsAO, format("Successfully authenticate user: %s", admin.login.toUpperCase()),
+            adminInfo = getInfo(systemLogsAO, format("Successfully authenticate user: %s", "NotAuthorized"),
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/SystemLoggingTest.java]: Why did we change this to "NotAuthorized"?

**Review by kamyshova — CHANGES_REQUESTED** [review]

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/SystemLoggingTest.java (hunk 4 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/SystemLoggingTest.java]: Changes have been removed.

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/NATGatewayAO.java (hunk 2 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/NATGatewayAO.java]: Done

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/LogAO.java (hunk 1 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/LogAO.java]: Done

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/SshPermissionTest.java (hunk 3 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/SshPermissionTest.java:132]: This method is not needed here

**Review by kamyshova — APPROVED** [review]

**Pull request merged**